### PR TITLE
i18n(ko-KR): translate `css-and-tailwind.mdx`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

translate `css-and-tailwind.mdx`

### CODE REVIEW

File: docs/src/content/docs/ko/guides/css-and-tailwind.mdx [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+
+   ```js
+   // astro.config.mjs
+   import { defineConfig } from 'astro/config';
+   import starlight from '@astrojs/starlight';
+
+   export default defineConfig({
+   	integrations: [
+   		starlight({
+   			title: 'Docs With Custom CSS',
```
Comment (delucis) [inline docs/src/content/docs/ko/guides/css-and-tailwind.mdx:32]: We could probably also translate the title text here like you did for the Tailwind example below.

**Review by delucis — COMMENTED** [review]
Looks great! Just noticed one detail, but otherwise, looks perfect!

I’m always really happy to see the theme editor in a new language ☺️ 

Starlight color theme editor with the UI translated to Korean

**Comment by jsparkdev** [comment]
#581 fixed this issue.

File: docs/src/content/docs/ko/guides/css-and-tailwind.mdx (hunk 1 above)
Comment (jsparkdev) [reply docs/src/content/docs/ko/guides/css-and-tailwind.mdx]: Thanks for the good point!

**Review by delucis — APPROVED** [review]
Perfect, thank you!

**Pull request merged**